.travis.yml

```diff
@@ -88,6 +88,8 @@ script:
         . scripts/get-libjvm-path.sh || return $?;
         export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$JNI_LIBDIR";
         make func-test V=1;
+        make VERBOSE=1 pytest-self-check;
+        make VERBOSE=1 pytest-check;
```

Comment:
Tip: If you want a verbose build with `autotools` you should use `V=1`, and the `VERBOSE=1` is for `cmake`.

Reply:
Oh, thanks for the tip. Otherwise as I saw the output, it does not add any other information for us. So I would remove this VERBOSE=1 from here, is it ok?
Otherwise when we want to get some debug info from failing tests we can run pytest locally with "python -m pytest ... -lvs --loglevel=debug [some test]". This information also can be public, I mean I can extend our wiki page in GitHub with some useful pytest commands, or can add a Readme.md to tests/pytest-framework/. What do you think?

Reply:
I agree to remove the verbosity.

I did not try the `autotool` version, but in case of `cmake` (+`Make`) the output of the pytest is still printed, what is not printed that what command is used to start pytest.